micromamba/tests/test_env.py

```diff
@@ -484,3 +484,33 @@ def test_env_update_empty_base(tmp_home, tmp_root_prefix, tmp_path):
     packages = helpers.umamba_list("-p", env_prefix, "--json")
     assert any(package["name"] == "xtensor" for package in packages)
     assert any(package["name"] == "python" for package in packages)
+
+
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+@pytest.mark.parametrize("json_flag", [None, "--json"])
+def test_env_export_with_pip(tmp_home, tmp_root_prefix, tmp_path, json_flag):
```

Comment:
Is this test sufficient, or should we test many more cases?

Reply:
LGTM.

Just to make sure - you checked that the json/yaml looks beautiful in the output?
I think tests don't check this at all.

Reply:
Yes, the output is properly indented.

For now, it is hard to come up with tests assessing the proper indentation since `helpers.run_env` is returning a dictionary when `--json` is passsed.

Also, I think we should rather introduce something systematic regarding such tests.